index.html

```diff
@@ -80,7 +80,109 @@ <h1 class="mn__title">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+        <div class="proj__container" id="proj__2">
+        <img class="proj__img" src="img/Work 2.webp" alt="1st project">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+        <div class="proj__container" id="proj__3">
+        <img class="proj__img" src="img/Work 3.webp" alt="1st project">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+        <div class="proj__container" id="proj__4">
+        <img class="proj__img" src="img/Work 4.jpg" alt="1st project">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+        <div class="proj__container" id="proj__5">
+        <img class="proj__img" src="img/Work 5.webp" alt="1st project">
+        <div class="proj__list">
+        <h3 class="proj__title">Multi-Post Stories Gain+Glory</h3>
+        <ul class="proj__lang">
+          <li class="bt"><a href="#">Ruby on rails</a></li>
+          <li class="bt"><a href="#">CSS</a></li>
+          <li class="bt"><a href="#">JavaScript</a></li>
+          <li class="bt"><a href="#">HTML</a></li>
+        </ul>
+        <button class="see__proj__btn">See Project</button>
+         </div>
+        </div>
+        <div class="proj__container" id="proj__6">
+        <img class="proj__img" src="img/Work 6.jpg" alt="1st project">
```

Comment:
- [ ] While you did a good job implementing the required images  :+1:  it seems that the last one doesn't show up. One reason could be that the src has an error, instead of calling `img/Work_6.jpg` you should use `img/work_6.jpg` 

Here you can see the error

![Screenshot from 2023-01-12 19-21-17](https://user-images.githubusercontent.com/105117832/212208933-41b1daf9-9194-4677-82c9-204ae0c67aaa.png)